src/data/markdown/docs/02 javascript api/07 k6-experimental/03 websockets/10 WebSocket/15-WebSocket-close--.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'WebSocket.close([code])'
+excerpt: 'Close the WebSocket connection.'
+---
+
+Close the WebSocket connection.
+
+| Parameter       | Type     | Description                                  |
+| --------------- | -------- | -------------------------------------------- |
+| code (optional) | number   | WebSocket status code. (default: 1000)       |
```

Comment:
Maybe link to the [supported WS close codes list](https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent/code)?